exercises/luhn/uLuhnExample.pas

```diff
@@ -0,0 +1,39 @@
+unit uLuhn;
+
+interface
+
+type
+  TLuhn = class
+    class function Valid(AInp : string) : boolean;
+  end;
+
+implementation
+
+uses
+  System.Regularexpressions, System.SysUtils, System.StrUtils;
+
+{ TLuhn }
+
+class function TLuhn.Valid(AInp: string): boolean;
+var
+  s : string;
+  i, R, P: int64;
+begin
+  R := 0;
+  s := ReverseString(TRegEx.Replace(AInp, ' ', ''));
```

Comment:
I think this will accomplish the same thing.
```pascal
s := ReverseString(AInp.Replace(' ', ''));
```

Reply:
The luhn equation states that we must move from right to left through the string.  I think this is why other solutions are reversing the input string.  Be sure to review https://en.wikipedia.org/wiki/Luhn_algorithm.

Don't worry about what the case descriptions say so much.  Just input the prescribed input and expect the expected response.